### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by Arcoders**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         body{
             background-color : orange; 
             color: white;
-            border : solid 10px skyblue;
+            border : solid 10px greenyellow;
```
Comment (HenryNovoa) [inline index.html:12]: insert random comment here

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
         body{
```
Comment (HenryNovoa) [inline index.html:9]: MOVE STYLES TO FILES!!

Comment (Arcoders) [reply index.html:9]: Okey! and you ?

Comment (HenryNovoa) [reply index.html:9]: Ok, i got it

**Pull request merged**